### PR TITLE
Adding Aptos Labs Case Study

### PR DESCRIPTION
**Pull request by aaronkao**
Adding https://docs.google.com/document/d/1W2hygXbk10ySWwsxMG1P0eqsaJyaTPcY2FCHPVfmqfo/edit

### CODE REVIEW

**Review by aaronkao — COMMENTED** [review]
A few questions and comments. 

1. Is the proserve team our proserve team or theirs?
2. There are some quotes and benefits sections that are missing because we don't have the components to do it. We should put it in after https://github.com/pulumi/docs/pull/12322 is complete

File: content/case-studies/aptos.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - label: Next Steps
+      anchor: next-steps
+
+aliases:
+    - /case-studies/aptos-labs
+---
+
+## About Aptos Labs
+
+Aptos Labs, co-founded by Mo Shaikh and Avery Ching, is dedicated to creating better network tooling and seamless usability to bring the benefits of decentralization to the masses. Having now raised over $400M, Aptos Labs is backed by top-flight investors, including a16z, Jump Crypto, Binance Labs, Dragonfly, PayPal Ventures, Andreessen Horowitz and Franklin Templeton Investments. Learn more at: https://aptoslabs.com/
```
Comment (interurban) [inline content/case-studies/aptos.md:31]:
```suggestion
Aptos Labs, co-founded by Mo Shaikh and Avery Ching, is dedicated to creating better network tooling and seamless usability to bring the benefits of decentralization to the masses. Having now raised over $400M, Aptos Labs is backed by top-flight investors, including a16z, Jump Crypto, Binance Labs, Dragonfly, PayPal Ventures, Andreessen Horowitz and Franklin Templeton Investments. Learn more at: [https://aptoslabs.com/](https://aptoslabs.com)
```

File: content/case-studies/aptos.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+The migration to Pulumi accelerated development at Aptos Labs and refocused their engineering team on writing and shipping differentiated code while spending less time on maintenance and duplicate efforts. Pulumi has made it easier for internal and external engineers to engage with infrastructure as code in the language of their choice, further accelerating development.
+
+## Benefits of the Pulumi Platform {#benefits}
+
+As part of migrating to Pulumi, the professional services team converted 13 modules at Aptos Labs across dev, test, and production environments for multiple cloud providers. The team used Pulumi to configure all aspects of the Aptos Labs landing zone for AWS and Google Cloud accounts including IAM roles, VPCs, DNS, secrets management, container registries, compute, and storage accounts. They also used Pulumi for advanced Kubernetes management including setting up Google Kubernetes Engine (GKE) clusters and configuring networking and observability for each cluster.
```
Comment (interurban) [inline content/case-studies/aptos.md:49]:
```suggestion
As part of migrating to Pulumi, the professional services team converted 13 modules at Aptos Labs across their dev, test, and production environments that spanned multiple cloud providers. The team used Pulumi to configure all aspects of the Aptos Labs landing zone for AWS and Google Cloud accounts including IAM roles, VPCs, DNS, secrets management, container registries, compute, and storage accounts. They also used Pulumi for advanced Kubernetes management including setting up Google Kubernetes Engine (GKE) clusters and configuring networking and observability for each cluster.
```

File: content/case-studies/aptos.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+With the successful migration from Terraform to Pulumi, Aptos Labs gained access to a more robust and flexible toolset. As a result, Aptos Labs is well-positioned to achieve a more efficient, scalable, and modern infrastructure management approach. Aptos Labs has also allowed engineers to reduce the cognitive burden of understanding a domain-specific language and work in the languages used for critical business functions. With Pulumi, Aptos Labs can now build usable and shareable software abstractions that encapsulate best practices for cloud infrastructure.
+
+## What's Next for Aptos Labs? {#next-steps}
+
+Aptos Labs and Pulumi plan to build on this success to advance improvements in their infrastructure management even further. These efforts will continue to optimize the structure and management of projects at Aptos Labs to achieve greater efficiency, scalability, and consistency.
+
+### Modularization of Projects
+
+Aptos Labs' projects are structured to combine multiple concerns within a single stack. This approach results in managing network, cluster, and application-specific resources together, leading to complexities, redundancies, and excess costs.
```
Comment (interurban) [inline content/case-studies/aptos.md:73]: I might reword this; along the lines of "Aptos Labs' projects are structured to combine multiple concerns within a single stack. There are opportunities to optimize and modularize how projects are structured to reduce complexities, redundancies and lower their total cloud spend."

**Review by interurban — APPROVED** [review]
Left you a few suggestions.

**Pull request merged**